src/frontend/metrics/generateMetricsReport.ts

```diff
@@ -22,17 +24,18 @@ export default function generateMetricsReport({
       addToSet(countries, positionToCountries(lat, lon));
     }
   }
-
   return {
     type: 'metrics-v1',
     appVersion: packageJson.version,
     os,
     osVersion,
     screen: {width: screen.width, height: screen.height},
     ...(countries.size ? {countries: Array.from(countries)} : {}),
+    percentageOfNetworkAvailability:
+      getPercentageOfNetworkAvailability(observations),
   };
 }
 
-function addToSet<T>(set: Set<T>, toAdd: Iterable<T>): void {
+export function addToSet<T>(set: Set<T>, toAdd: Iterable<T>): void {
```

Comment:
Nitpick: IMO, this makes more sense as a separate utility. Could you move it to a separate file, or avoid using it in the test?

Reply:
yeah, I fixed it like this cause it seem weird that `positionToCountries` returns a `Set` instead of only one country. But seeing your `disputed territory` comment, now it makes sense...